src/components/ResumeUploadBtn.js

```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useState } from 'react'
+import { Button } from '../components/Button'
+import { QuestionHeading } from '../components/Typography'
+
+const resumeStyle = {
+  display: 'flex',
+  alignItems: 'center'
+};
```

Comment:
We use styled components instead of style objects. This can be changed to 
```suggestion
const ResumeContainer = styled.div`
  display: flex;
  alignItems: center;
`
```

Reply:
thanks for the comment, but it seems to behave differently now and the elements are no longer aligned if you have any ideas as to the cause? I found one person with a similar issue when googling but the fix was not applicable :(
![image](https://user-images.githubusercontent.com/37602670/100578114-72f3de00-3296-11eb-9833-a12f9f35a97a.png)

Reply:
Could you push up what you changed so I can take a look?

Reply:
Oh. It should be `align-items` not `alignItems`

Reply:
oooooh ok that fixed it thank you boss. and i'll try to push by tonight. almost done addressing all the comments here just tryna fix stuff up!

Reply:
resolved in latest commit!